lunatrace/bsl/backend/src/config.ts

```diff
@@ -67,9 +68,17 @@ export function getAwsConfig(): AwsConfig {
 }
 
 export function getWebhookConfig(): WebhookConfig {
+  const shouldDisableWebhookQueue = checkEnvVar('DISABLE_WEBHOOK_QUEUE', 'false');
+
+  const disableWebhookQueue = validateBooleanString(shouldDisableWebhookQueue);
+  if (disableWebhookQueue.error) {
```

Comment:
Nice check. Tiny nit and doesn't need fixing but I can see the golang code leaking through.  For JS I would probably just have that function itself throw and return a boolean instead of a wrapped boolean.